paasta_tools/contrib/paasta_update_soa_memcpu.py

```diff
@@ -214,11 +223,14 @@ def main(argv=None):
 
         serv['state'] = provisioned_state
         ticket_desc = (
-            "We suspect that {s}.{i} in {c} may be {o}-provisioned"
-            " as of {d}. It initially had {x} cpus, but we think"
+            "This ticket and CR have been auto-generated, and only requires action from your team to ship."
+            "\nWe suspect that {s}.{i} in {c} may have been {o}-provisioned"
+            " during the 1 week prior to {d}. It initially had {x} cpus, but we think"
             " it needs {y} cpus."
-            "\n- Dashboard: https://y.yelpcorp.com/{o}provisioned\n- Service"
-            " owner: {n}\n- Estimated monthly excess cost: ${m}"
+            "\n- Dashboard: https://y.yelpcorp.com/{o}provisioned?{superregion_param}&{service_param}&{instance_param}"
+            "\n- Service owner: {n}"
+            "\n- Estimated monthly excess cost: ${m}"
+            "\n\nFor more information:"
             "\n- Runbook: https://y.yelpcorp.com/rb-provisioning-alert"
             "\n- Alert owner: team-perf@yelp.com"
```

Comment:
Can you add even more? Can you add who is supposed to give a shipit, who is responsible for merging changes in, what happens after the change is merged?

Reply:
Hmm, yea. I thought adding the auto-generated line would be helpful but this is definitely still a new process. I can see why more info is better. Do you think this needs 2 people to respond (1 to shipit, 1 to merge) or can we clarify here that we expect the person who reviews the number/dashboard can both shipit and merge the jenkins branch?

Was there something more you were suggesting re: the who part?

I can see the recommended steps in-line being helpful here. More minor clarifications off the beaten path we can always add to the y/rb-provisioning-alert doc anytime.

Would an announcement in EPS that we're now automatically ticketing and CRing over-provisioned services to reduce costs, and quickly explain that merge process, and explain why each team has to do their part even though they are individually just drops in a bucket help increase traction?